package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "bookstore",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.15.0",
+    "react-scripts": "5.0.1",
+    "uui": "^1.0.7",
+    "uuid": "^9.0.0",
+    "web-vitals": "^2.1.4"
```

Comment:
- [ Required ] I think it would be nice if you install the `@reduxjs/toolkit` and `react-redux` node packages in your project so that they be listed as dependencies in `package.json`. Currently, your project imports methods from `@reduxjs/toolkit` yet it is not listed as a dependency in the package.json file, this has the potential to cause an error when running the project. You could install these node packages using the code listed below.

## Example code
```sh
$ npm install @reduxjs/toolkit
$ npm install react-redux
```